pandas/core/dtypes/dtypes.py

```diff
@@ -699,7 +699,7 @@ class DatetimeTZDtype(PandasExtensionDtype):
     ----------
     unit : str, default "ns"
         The precision of the datetime data. Currently limited
```

Comment:
I think we should consider deleting "Currently limited" since there are 4 possible options now.

Reply:
@Aloqeely thank you for your review.

How about

```
The precision of the datetime data. Only supports
``"s"``, ``"ms"``, ``"us"``, ``"ns"``.
```

as same as error message?

```py
if unit not in ["s", "ms", "us", "ns"]:
    raise ValueError("DatetimeTZDtype only supports s, ms, us, ns units")
```

Reply:
Yes sure, `Only supports` and `Valid options are` both sound good to me.